File: storio-content-resolver-annotations-processor/src/main/java/com/pushtorefresh/storio/contentresolver/annotations/processor/introspection/StorIOContentResolverTypeMeta.java
package com.pushtorefresh.storio.contentresolver.annotations.processor.introspection;

import com.pushtorefresh.storio.common.annotations.processor.introspection.StorIOTypeMeta;
import com.pushtorefresh.storio.contentresolver.annotations.StorIOContentResolverType;

import org.jetbrains.annotations.NotNull;

public class StorIOContentResolverTypeMeta extends StorIOTypeMeta<StorIOContentResolverType, StorIOContentResolverColumnMeta> {

    public StorIOContentResolverTypeMeta(
            @NotNull String simpleName,
            @NotNull String packageName,
            @NotNull StorIOContentResolverType storIOType) {
        super(simpleName, packageName, storIOType);
    }
}

File: storio-content-resolver-annotations-processor/src/main/kotlin/com/pushtorefresh/storio/contentresolver/annotations/processor/introspection/StorIOContentResolverTypeMeta.kt
package com.pushtorefresh.storio.contentresolver.annotations.processor.introspection

import com.pushtorefresh.storio.common.annotations.processor.introspection.StorIOTypeMeta
import com.pushtorefresh.storio.contentresolver.annotations.StorIOContentResolverType

class StorIOContentResolverTypeMeta(
        simpleName: String,
        packageName: String,
        storIOType: StorIOContentResolverType)
    : StorIOTypeMeta<StorIOContentResolverType, StorIOContentResolverColumnMeta>(
        simpleName,
        packageName,
        storIOType)